src/scanner-cli.ts

```diff
@@ -18,15 +18,16 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 import { spawn } from 'child_process';
+import fs from 'fs';
```

Comment:
why not using `fs-extra` which is already imported? it bundles native `fs` methods

Reply:
thanks, updated